src/plugin.js

```diff
@@ -175,17 +174,16 @@ class VR extends Plugin {
           }
         }
       }
-      geometry.scale(-1, 1, 1);
 
       this.movieGeometry = new THREE.BufferGeometry().fromGeometry(geometry);
       this.movieMaterial = new THREE.MeshBasicMaterial({ map: this.videoTexture, overdraw: true, side: THREE.BackSide });
 
       this.movieScreen = new THREE.Mesh(this.movieGeometry, this.movieMaterial);
-      this.movieScreen.rotation.y = -Math.PI / 2;
+      this.movieScreen.scale.x = -1;
+      this.movieScreen.quaternion.setFromAxisAngle({x: 0, y: 1, z: 0}, -Math.PI / 2);
```

Comment:
What do these changes do?

Reply:
Copied from `360` projection. This transforms the geometry so that it maps the texture correctly. The geometry is rotated by pi/2 and flipped.

The version on master is completely broken, resigning in a blank canvas, IIRC. You can test that with a 360_LR or 360_TB video.

Reply:
> You can test that with a 360_LR or 360_TB video. 

`coriolis_rect.webm` is one ;)